charts/ocis/values.yaml

```diff
@@ -4,9 +4,9 @@ image:
   # -- Image repository
   repository: owncloud/ocis
   # -- Image tag. Defaults to the chart's appVersion.
-  tag: "latest"
+  tag: ""
   # -- Image sha / digest (optional).
-  sha: "13e5447c5e11de9068fbab07f362a793fbdc32edd8ad14eff0db1d21dfa2a5b8" # oCIS as of 5.0.0-alpha.4
+  sha: ""
```

Comment:
Is it not possible to pin the alpha 5 version?

Reply:
We actually didn't pin the digest for tags so far. We had it on `next` because we used some unreleased oCIS "latest" version since we didn't have releases yet.

Reply:
But with the change here we would then neither have a tag nor a sha.

Reply:
tag is not needed, see https://github.com/owncloud/ocis-charts/pull/442#discussion_r1417078997